src/debug-registry.h

```diff
@@ -12,7 +12,8 @@ typedef struct {
     const llvm::object::ObjectFile *obj;
     llvm::DIContext *ctx;
     int64_t slide;
-} objfileentry_t;
+    std::map<uintptr_t, StringRef, std::greater<size_t>> *symbolmap;
```

Comment:
out of curiosity, why use the difference comparsion operator?

Reply:
It has the right boundary behavior when used with `std::lower_bound` to find the nearest entry before (or equal to) the provided key.